packages/jest-matchers/src/__tests__/spyMatchers-test.js

```diff
@@ -114,34 +101,27 @@ describe('toHaveBeenCalledTimes', () => {
 });
 
 [
-  ['lastCalledWith', 'jest.fn'],
-  ['toBeCalledWith', 'jest.fn'],
-  ['toHaveBeenCalledWith', 'jasmine.createSpy'],
-  ['toHaveBeenCalledWith', 'jest.fn'],
-  ['toHaveBeenLastCalledWith', 'jasmine.createSpy'],
-  ['toHaveBeenLastCalledWith', 'jest.fn'],
-].forEach(([calledWith, mockName]) => {
-  const getFunction = () => {
-    return mockName === 'jest.fn' ? jest.fn() : jasmine.createSpy('fn');
-  };
-
-  test(`${calledWith} works with ${mockName} when not called`, () => {
-    const fn = getFunction();
+  'lastCalledWith',
+  'toHaveBeenCalledWith',
+  'toHaveBeenLastCalledWith',
+].forEach(calledWith => {
+  test(`${calledWith} works when not called`, () => {
+    const fn = jest.fn();
     jestExpect(fn).not[calledWith]('foo', 'bar');
 
     expect(() =>
       jestExpect(fn)[calledWith]('foo', 'bar'),
     ).toThrowErrorMatchingSnapshot();
   });
 
-  test(`${calledWith} works with ${mockName} and no arguments`, () => {
-    const fn = getFunction();
+  test(`${calledWith} works with no arguments`, () => {
+    const fn = jest.fn();
     fn();
     jestExpect(fn)[calledWith]();
   });
 
-  test(`${calledWith} works with ${mockName} and arguments that don't match`, () => {
-    const fn = getFunction();
+  test(`${calledWith} works when arguments that don't match`, () => {
```

Comment:
Not sure if this was mistakenly changed to `when`; it was `with` previously, like the other tests here.